controllers/controllers.go

```diff
@@ -75,14 +78,27 @@ func (r *instanceReconciler) ensureInstanceIsUpToDate(instance *instances.Instan
 // instanceFromNode returns an instance object for the given node. Requires a username that can be used to SSH into the
 // instance to be annotated on the node.
 func (r *instanceReconciler) instanceFromNode(node *core.Node) (*instances.InstanceInfo, error) {
-	if node.Annotations[UsernameAnnotation] == "" {
+	usernameAnnotation := node.Annotations[UsernameAnnotation]
```

Comment:
should we avoid "api call stutters" like Aravindh mentioned yesterday? Maybe you could keep this line how it was it was previously

Reply:
Don't think that is applicable here. Also `usernameAnnotation` is used in multiple places so I think it warrants a variable.